app/views/mission/missions/edit.html.slim

```diff
@@ -1,3 +1,5 @@
-h1 = '修改任务'
+.page-form
+  .page-header
```

Comment:
And you can learn to use a helper method called `page_header` which is written by myself. You can find its usage in `missions/show` page.